src/components/BookList.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] Kindly don't disable your linters and try to fix your linters manually 👍 because linters prevents your from introducing those errors which can break your app 😨